build/azure-pipelines/build-common.yml

```diff
@@ -10,6 +10,10 @@ parameters:
 - name: VersionTag
   type: string
   default:  $(Build.BuildNumber)
+- name: BinaryName
+  type: string
+  default:
```

Comment:
don't put a default attribute here since it's really required to be provided by the importer

Reply:
Removed the default attribute